src/Microsoft.DotNet.XHarness.Android/Execution/BaseReportManager.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DotNet.XHarness.Android.Execution
+{
+    class BaseReportManager
+    {
+        public ILogger Logger { get; }
```

Comment:
Should this be `protected`?

```suggestion
        protected ILogger Logger { get; }
```

Reply:
Actually.. Better pattern is not to reuse dependencies from a parent but have your own copy - the idea is that you don't know what the parent might do with dependencies and so you create your own `private` field `_log` and work with it as this is now your class' implementation.